src/main/scala/EnsimePlugin.scala

```diff
@@ -228,8 +228,9 @@ object EnsimePlugin extends AutoPlugin {
 
     val updateReports = ensimeMegaUpdate.run
 
+    val javaH = (ensimeJavaHome).gimme
     val scalaCompilerJars = ensimeScalaJars.run.toSet
-    val serverJars = ensimeServerJars.run.toSet -- scalaCompilerJars
+    val serverJars = ensimeServerJars.run.toSet -- scalaCompilerJars + javaH / "tools.jar"
```

Comment:
Implementing new launcher for the Vim client taking advantage of `:ensime-server-jars` and I just came across this—on an Oracle JDK install on OS X (1.7 and 1.8), `tools.jar` is in `$JAVA_HOME/lib/`, not at the root of `JAVA_HOME`. Is this consistent across JDKs/platforms?

Our old classpath-building code also assumed it's under `lib`, so this looks like a mistake. If so I can make a PR if no one beats me to it.